appveyor.yml

```diff
@@ -0,0 +1,22 @@
+# This is the one with Docker
```

Comment:
Does this fix the Appveyor test breakage that github has been complaining about?

Reply:
It will once I work with @ericxtang to get a few more CI parameters tweaked :)